cloud/src/autogluon/cloud/scripts/train.py

```diff
@@ -3,11 +3,13 @@
 import shutil
 import yaml
 
-import autogluon.text
-
 from autogluon.tabular import TabularPredictor, TabularDataset, FeatureMetadata
-from autogluon.text import TextPredictor
 from autogluon.vision import ImagePredictor
+from autogluon.text import TextPredictor
+
+# Not following import style because importing autogluon.tet before other module would cause segfault
+# https://github.com/awslabs/autogluon/issues/2042
+import autogluon.text
```

Comment:
redundant text import? Don't you import TextPredictor right above this line?

Reply:
So the import of text module itself is for determining how to correctly import MultimodalPredictor as we moved it from text to multimodal module in 0.5 release. The import of TextPredictor sort of just align with other module's import. We can of course only import autogluon.text and use autogluon.text.TextPredictor. Not sure what's the convention

Reply:
I'm just confused why we are doing:

```
from autogluon.text import TextPredictor
import autogluon.text
```

Instead of just:

```
from autogluon.text import TextPredictor
```